src/Native/Unix/System.Security.Cryptography.Native/pal_ecc_import_export.c

```diff
@@ -214,23 +269,23 @@ int32_t CryptoNative_GetECCurveParameters(
         goto error;
 
     group = EC_KEY_get0_group(key); // curve
-    if (!group) 
+    if (!group)
         goto error;
 
     curveMethod = EC_GROUP_method_of(group);
-    if (!curveMethod) 
+    if (!curveMethod)
         goto error;
 
     *curveType = MethodToCurveType(curveMethod);
-    if (*curveType == Unspecified) 
+    if (*curveType == Unspecified)
         goto error;
 
-    // Extract p, a, b
+        // Extract p, a, b
```

Comment:
Couldn't find a good solution to avoid this one. There are three options:

1. ignore this nit
2. use brackets around previous line `goto error` (that will make it inconsistent with rest of the file)3. `clang-format off/on` comment around this comment, which seems like an overkill